src/main.cpp

```diff
@@ -1,8 +1,23 @@
-#include <iostream>
-#include "CourseDashboard.hpp"
-
-int main()
-{
-    std::cout << "Course Dashboard" << std::endl;
-    return 0;
-}
+  #include "train.hpp"
+  #include "stud.hpp"
+  #include <iostream>
+  #include "CourseDashboard.hpp"
+
+
+  int main()
+  {
+      std::cout << "Course Dashboard" << std::endl;
+      StudentData database;
+      database.addUser("Wiktor","Arakis14","github.com/test1","weekend group");
+      database.addUser("Jan","kowal1234","github.com/test2","evening group");
+      database.addUser("Test","something ","anything","test");
+      std::cout<<"=============================="<<std::endl;
+      database.viewUsers();
+      database.deleteUser(1);
```

Comment:
from this point of view, what is 1? When I will do a mass import of students which of them will have number 1? It's not obvious how should I know this number.